src/button/README.md

```diff
@@ -79,7 +79,7 @@ There are a number of visual appearances that the `vscode-button` can have. The
 <vscode-button appearance="primary">Button Text</vscode-button>
 <vscode-button appearance="secondary">Button Text</vscode-button>
 <vscode-button appearance="icon">
-	<span class="codicon codicon-check"></span>
```

Comment:
Are these just from prettier?

Reply:
Yep, just changed tab widths from 4 spaces to 2 spaces